index.html

```diff
@@ -219,6 +220,36 @@ <h3>Skills</h3>
                 </li>
             </ul>
         </section>
+
+        <section class="contact-me" id="contact-me">
+            <div class="contact-container">
+                <div class="contact-content">
+                    <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+                    coding. I’d love to help with it
+                    </p>
+                    <form action="/" method="post">
+                        <ul>
```

Comment:
- [ ] Nice work implementing the form, it looks really good 👏 However it states in the project requirement that data should be collected by using the [Formspree service](https://formspree.io/html). Therefore I suggest you:😊⤵️
- 1. Add the action attribute to your form tag and its value should be similar to https://formspree.io/f/{form_id} where {form_id} is the id given you when creating an account on [formspree service](https://formspree.io/html).
- 2. Add the method attribute to your form tag and its value equals POST

- [ ] Hint: Click [here](https://youtu.be/sVPlb5C3oQ8) to watch this video to help you how to integrate the form spree with your form 😊💯